src/helpers/messages.js

```diff
@@ -57,23 +57,23 @@ function getUnansweredItemsWarning(scope, runner, sync) {
     var itemsCountMessage = '';
 
     if (scope === 'section' || scope === 'testSection') {
-        if (unansweredCount === 0) {
-            itemsCountMessage = __('You answered all %s question(s) in this section', stats.questions.toString());
-        } else {
-            itemsCountMessage = __(
-                'You answered only %s of the %s question(s) in this section',
-                stats.answered.toString(),
-                stats.questions.toString()
-            );
-        }
+        itemsCountMessage = `<b>${__('You are about to leave this section.')}</b><br><br>`
+        itemsCountMessage += __(
+            'You answered %s of %s question(s) for this section of the test.',
+            stats.answered.toString(),
+            stats.questions.toString()
+        );
         if (flaggedCount) {
             itemsCountMessage += `, ${__('and flagged %s of them', flaggedCount.toString())}`;
         }
     } else if (scope === 'test' || scope === 'testWithoutInaccessibleItems') {
-        if (unansweredCount === 0) {
-            itemsCountMessage = __('You answered all %s question(s) in this test', stats.questions.toString());
-        } else {
-            itemsCountMessage = __('You have %s unanswered question(s)', unansweredCount.toString());
+        itemsCountMessage = `<b>${__('You are about to submit the test.')}</b><br><br>`
+        if (unansweredCount > 1) {
+            itemsCountMessage += __('There are %s unanswered questions. You will not be able to return to this test after you submit your answers.', stats.answered.toString());
+        } else if (unansweredCount === 1) {
+            itemsCountMessage += __('There is %s unanswered question. You will not be able to return to this test after you submit your answers.', stats.answered.toString());
+        } else if (unansweredCount === 0) {
+            itemsCountMessage += __('You will not be able to return to this test after you submit your answers.');
```

Comment:
`flaggedCount`-condition message `'and you flagged %s item(s) that you can review now'` is going to look awkward now.

It should probably also start from a capital letter since we define all message parts as separate sentences.

Reply:
When product decide this, don't take into account flagged. I create this issue for handle this: https://oat-sa.atlassian.net/browse/BSA-210

Reply:
I fixed, not at least don't looks awkward, but probably still review this messages in that new issue